installers/go-agent/release/agent.sh

```diff
@@ -115,12 +115,14 @@ if [ "$PRODUCTION_MODE" == "Y" ]; then
     if [ -d /var/log/${SERVICE_NAME} ]; then
         LOG_DIR=/var/log/${SERVICE_NAME}
     else
-	    LOG_DIR=$AGENT_WORK_DIR
+	    LOG_DIR=$AGENT_WORK_DIR/logs
     fi
 else
-    LOG_DIR=$AGENT_WORK_DIR
+    LOG_DIR=$AGENT_WORK_DIR/logs
 fi
 
+mkdir -p "${LOG_DIR}"
+
 STDOUT_LOG_FILE=$LOG_DIR/${SERVICE_NAME}-bootstrapper.out.log
```

Comment:
There is a `LOG_FILE` variable that is exported unnecessarily and stop-agent.sh also sets it unnecessarily. I did a `git grep -w "LOG_FILE"` in the project and found no usages in the code, except for `JavaApplicationStub64`. I think we can remove it.